sources/postgres/pgdump.go

```diff
@@ -732,6 +732,12 @@ func getRows(conv *internal.Conv, vll []*pg_query.Node, n *pg_query.InsertStmt)
 						// high priority (it isn't right now), then consider preserving int64
 						// here to avoid the int64 -> string -> int64 conversions.
 						values = append(values, strconv.FormatInt(int64(c.Integer.Ival), 10))
+					case *pg_query.Node_Float:
+						values = append(values, c.Float.Str)
+					case *pg_query.Node_Null:
```

Comment:
For rows of the form (NULL, 14.67), the pg_query node type is Node_Null and Node_Float respectively. 

Data dumped with quotes ('NULL', '14.67') would have worked as it would get classified as Node_String_ but without quotes, the pg_query parser assigns a different type to the data.

Reply:
what is the different type that gets assigned ? Add this description in comments within the code

Reply:
Done